vllm/model_executor/models/llama.py

```diff
@@ -87,14 +87,17 @@ def __init__(
             quant_config=quant_config,
             prefix=f"{prefix}.down_proj",
         )
-        self.use_fp8 = isinstance(quant_config, Fp8Config)
+        self.use_fp8 = isinstance(quant_config, Fp8Config) \
+                       if is_hip() and not is_navi4x() else False
         if hidden_act != "silu":
             raise ValueError(f"Unsupported activation: {hidden_act}. "
                              "Only silu is supported for now.")
         self.act_fn = SiluAndMul()
 
     def forward(self, x):
-        if is_hip() and x.shape[0] == 1 and x.shape[1] == 1:
+        # Navi4x is diff from other HIP devices by using e4m3fn fp8 format
+        if is_hip() and not is_navi4x() \
```

Comment:
This is not about fp8, it's general for anything ROCm

Reply:
Will update.